src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,41 @@
+REM 
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
```

Comment:
Agreed -- for now.  But I think we will very soon want to allow humans to run a subset of this script to do trial runs of PR's.  We don't need to design for that now, but it is worth keeping in mind.